src/Network/Network/VirtualNetwork/NewAzureVirtualNetworkCommand.cs

```diff
@@ -66,6 +66,12 @@ public class NewAzureVirtualNetworkCommand : VirtualNetworkBaseCmdlet
             HelpMessage = "The list of Dns Servers")]
         public string[] DnsServer { get; set; }
 
+        [Parameter(
+             Mandatory = false,
+             ValueFromPipelineByPropertyName = true,
+             HelpMessage = "FlowTimeout value of the virtual network should be between 4 and 30 mins (inclusive)")]
```

Comment:
```suggestion
             HelpMessage = "FlowTimeout value of the virtual network should be between 4 and 30 minutes (inclusive)")]
```
Please avoid abbreviation

Reply:
Actually, I have no idea what FlowTimeout is. Is it a terminology in virtual network? Could you add detail descriptions about this parameter in HelpMessage?